go/libcmdline/cmdline.go

```diff
@@ -352,6 +367,18 @@ func (p *CommandLine) PopulateApp(addHelp bool, extraFlags []cli.Flag) {
 			Name:  "api-timeout",
 			Usage: "set the HTTP timeout for API calls to the keybase API server",
 		},
+		cli.StringFlag{
+			Name:  "tor-mode",
+			Usage: "set TOR mode to be 'leaky', 'none', or 'strict'. 'none' by default",
```

Comment:
we can do something like `keybase help tor` ?

On Fri, Nov 6, 2015 at 11:57 AM, oconnor663 <notifications@github.com>
wrote:

> In go/libcmdline/cmdline.go
> <https://github.com/keybase/client/pull/1290#discussion_r44159868>:
>
> > @@ -352,6 +367,18 @@ func (p *CommandLine) PopulateApp(addHelp bool, extraFlags []cli.Flag) {
> >  			Name:  "api-timeout",
> >  			Usage: "set the HTTP timeout for API calls to the keybase API server",
> >  		},
> > +		cli.StringFlag{
> > +			Name:  "tor-mode",
> > +			Usage: "set TOR mode to be 'leaky', 'none', or 'strict'. 'none' by default",
>
> Is there space to document that "strict tries to be anonymous even to the
> Keybase server, leaky uses Tor for your connection but identifies you to
> the server"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1290/files#r44159868>.
>